### PR TITLE
Cross encoder 

### PR DESCRIPTION
**Pull request by Micost**
Add cross encoder class with rank and predict functions

### CODE REVIEW

File: src/kimchima/cmds/auto_cli.py [hunk 1]
```diff
@@ -41,3 +41,45 @@ def auto(args):
         model = ModelFactory.auto_model(pretrained_model_name_or_path=args.model_name_or_path)
         print(model.config)
 
+class CommandCrossEncoder:
```
Comment (Aisuko) [inline src/kimchima/cmds/auto_cli.py:44]: This sub-command can works right?

File: src/kimchima/pkg/cross_encoder_factory.py [hunk 2]
```diff
@@ -0,0 +1,282 @@
+from transformers import AutoModelForSequenceClassification, AutoTokenizer, AutoConfig
```
Comment (Aisuko) [inline src/kimchima/pkg/cross_encoder_factory.py:1]: I'd like to add a linter to check the styles of the  import libs. However, it should be not an issue in this PR.

Comment (Aisuko) [reply src/kimchima/pkg/cross_encoder_factory.py:1]: I want to mentioned, if we use the code from other repos. We should add

"""
Original code from: URL
Adapted by: Your name
Date: adopt date
"""

We acknowledge their work, and hope they do same to others

Comment (Micost) [reply src/kimchima/pkg/cross_encoder_factory.py:1]: Add comments for original code

**Review by Aisuko — APPROVED** [review]
LGTM

**Pull request merged**